### ISSUE TITLE
About feature extraction history

### ISSUE THREAD

**Issue by Dovid308** [Yuhan-Shen/ProTAS#7]
Hi!
I was trying to understand how the different dataset features were extracted. 
Have you used the actual feature shared in the read me for the training? 

Essentially I suppose GTEA ones where precedently (should be i3d two stream 21 window).  The same happen for Egoprocel in the contest of FACT paper (only rgb probably not causal but im not sure https://github.com/ZijiaLewisLu/CVPR2024-FACT/issues/2 by this repo https://github.com/ZijiaLewisLu/action-recognition-pytorch). 

So as far as this paper you extracted personally the egoper ones. Are you using the same idea of Egoprocel, so only one RGB stream? 

As far as this in the paper you speak about a 32 frame history " We extract 2048-dim features on a window of 32 history frames using I3D pretrained on Kinetics [10]" while in  #4  you speak about shifting the label of 10. This look like a cotradiction to me.  

Could you try to clarify this to me? I'm trying to decide whether to rely on your provided features or re-extract them, so any detail would be really helpful.

Tank you in advance!
